WhatsNew.Cli/CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+  - Go to **Settings** on your repo.
+  - Select **Secrets**
+  - Add "OpsoKey" as a **Repository Secret**.
+
+## Build and run
+
+There are 2 supported approaches to running the tool locally, as outlined in the following sections. Choose the approach that best suits your needs.
+
+### Run from the generated DLL
+
+1. Build the *WhatsNew.Cli* project.
+1. Navigate to the *WhatsNew.Cli* project's *bin\Debug\net5.0* directory.
+1. Run the appropriate `dotnet whatsnew.dll` command from the aforementioned directory. For example:
+
+    ```bash
+    dotnet whatsnew.dll -h
+    ```
+
+### Run as a .NET global tool
+
+1. Build the *WhatsNew.Cli* project. Doing so packages the project as a [.NET global tool](https://docs.microsoft.com/dotnet/core/tools/global-tools), as represented by the *.nupkg* file in the project's *nuget* directory.
+1. Run the following command from the project's root directory to install the global tool:
+
+    ```bash
+    dotnet tool install -g --add-source nuget dotnet-whatsnew
+    ```
+
+1. Run the following command to confirm installation of the global tool:
+
+    ```bash
+    dotnet whatsnew -h
+    ```
+
+## Build and deploy (CI/CD)
+
+### Build
+
+Build automation is handled by Azure Pipelines via the [dotnet-docs-tools build pipeline](https://dev.azure.com/mseng/TechnicalContent/_build?definitionId=10534). A YAML representation of this build pipeline is stored in the *azure-pipelines.yml* file.
+
+The build pipeline contains a `trigger` section which determines the branch names/patterns and paths that trigger a build. For example:
+
+```yml
+trigger:
+  branches:
+   include:
+     - master
+     - scottaddie/*
+  paths:
+    include:
+      - whatsnew/*
+    exclude:
+      - whatsnew/src/WhatsNew.Cli/CONTRIBUTING.md
+      - whatsnew/src/WhatsNew.Cli/README.md
+```
+
+With the preceding configuration:
+
+- A commit to *master* or a branch prefixed with *scottaddie/* triggers the build.
+- Any file in the *whatsnew* directory triggers a build, with the exception of the Markdown files listed in `exclude`.
+
+### Release
+
+Publishing of the NuGet package (containing the .NET global tool) is handled by the last task in the build pipeline. The package version number is determined by the *WhatsNew.Cli* project's `<Version>` element's value. If the publish is successful, the new package version appears in the Azure Artifacts [DotnetDocsTools feed](https://dev.azure.com/mseng/TechnicalContent/_packaging?_a=feed&feed=DotnetDocsTools%40Local).
+
+Complete the following steps to publish the package:
+
+1. Increment the version number in the project file's `<Version>` element and commit the change. Wait for the build to complete, and ensure that the change has been merged to *master*.
+1. Select the **Run pipeline** button at https://dev.azure.com/mseng/TechnicalContent/_build?definitionId=10534.
```

Comment:
Is this dev.azure.com link still valid with the move to the public repo?

Reply:
No, but I'll update that in a later PR. (We plan to update this to make it a GitHub Action. The action will pull the source and build it.)